lxd/images.go

```diff
@@ -3001,7 +3081,13 @@ func imageGet(d *Daemon, r *http.Request) response.Response {
 	// Get the image. We need to do this before the permission check because the URL in the permission check will not
 	// work with partial fingerprints.
 	var info *api.Image
+	effectiveProjectName := projectName
```

Comment:
`var effectiveProjectName string` would be clearer here no? Showing that its empty until its loaded from the DB below, rather than defaulting to the request project?